gettext_compile.py

```diff
@@ -8,6 +8,6 @@
 
 if __name__ == "__main__":
     do_compile("spyderlib")
-    do_compile("p_pylint", "spyderplugins")
-    do_compile("p_profiler", "spyderplugins")
-    do_compile("p_breakpoints", "spyderplugins")
+    do_compile("p_pylint", "spyderplugins/p_pylint")
+    do_compile("p_profiler", "spyderplugins/p_pylint")
+    do_compile("p_breakpoints", "spyderplugins/p_pylint")

```

Comment:
`gettext_compile.py ` runs fine, but `gettext_scan.py` gives:
```
Traceback (most recent call last):
  File "gettext_scan.py", line 10, in <module>
    do_rescan("spyderlib")
  File "/home/joseph/prog/spyder.git/gettext_helpers.py", line 46, in do_rescan
    do_rescan_files(files, modname, dirname)
  File "/home/joseph/prog/spyder.git/gettext_helpers.py", line 55, in do_rescan_files
    ]+files)
  File "/usr/lib/python3.4/subprocess.py", line 537, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.4/subprocess.py", line 859, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.4/subprocess.py", line 1457, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'pygettext'
```
Do I have to install `pygettext` from somewhere ?